src/Compiler/Service/BackgroundCompiler.fs

```diff
@@ -1370,16 +1370,10 @@ type internal BackgroundCompiler
             scriptClosureCache.Set(AnyCallerThread, options, loadClosure) // Save the full load closure for later correlation.
 
             let diags =
+                let flatErrors = options.OtherOptions |> Array.contains "--flaterrors"
```

Comment:
Now I'm wondering if this could be extracted to the environment (if it's not there yet). But it's not a blocker for this PR, I guess 🙂

Reply:
> Now I'm wondering if this could be extracted to the environment (if it's not there yet). But it's not a blocker for this PR, I guess 🙂

Yeah, in the whole TransparentCompiler (which this code is part of) there is still a lot left to do. But indeed not as part of this PR.